Applications/spire/source/book_view/quote_panel.cpp

```diff
@@ -23,20 +23,19 @@ QuotePanel::QuotePanel(const BookViewModel& model, Side side,
   auto label_layout = new QHBoxLayout();
   label_layout->setContentsMargins({});
   label_layout->setSpacing(0);
-  label_layout->addStretch(1);
   m_price_label = new QLabel(this);
-  m_price_label->setFixedWidth(scale_width(52));
-  m_price_label->setAlignment(Qt::AlignCenter);
+  m_price_label->setIndent(0);
+  m_price_label->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Expanding);
```

Comment:
```Expanding``` means that the label can grow or shrink based on the contents of the widget. We don't want this label constantly changing size based on the value, we want its size to remain stable. Review the policies found here:

http://doc.qt.io/qt-5/qsizepolicy.html#Policy-enum

Specifically you are likely to get better results if you use the ```ExpandingMinimum```. Use that policy for both price and size labels and put up a demo on Dropbox for review.

Reply:
I put the demo up.